docker-compose.rollkit.yml

```diff
@@ -1,12 +1,17 @@
 services:
-  beacond:
-    image: ghcr.io/rollkit/beacon-kit:v0.1.3
-    container_name: beacond
+  rollkit-evm-single:
+    image: ghcr.io/rollkit/rollkit-evm-single:v0.1.0
+    container_name: rollkit-evm-single
     volumes:
-      - ./testing/files:/beacon-kit/testing/files
-      - ./.tmp:/beacon-kit/.tmp
       - /var/run/docker.sock:/var/run/docker.sock
-    entrypoint: /beacon-kit/testing/files/entrypoint.sh
+    entrypoint: /usr/bin/entrypoint.sh
+    environment:
+      - EVM_ENGINE_URL=http://reth:8551
+      - EVM_ETH_URL=http://reth:8545
+      - EVM_JWT_SECRET=f747494bb0fb338a0d71f5f9fe5b5034c17cc988c229b59fd71e005ee692e9bf
+      - EVM_GENESIS_HASH=0x0a962a0d163416829894c89cb604ae422323bcdf02d7ea08b94d68d3e026a380
```

Comment:
Is this the has of the `eth-genesis.json` file?

Reply:
yes, will update it with the changes: https://github.com/rollkit/go-execution-evm?tab=readme-ov-file#reading-genesis-information